docs/msw/install.md

```diff
@@ -472,6 +476,18 @@ need to:
   (all wxWidgets applications use the base library).
 
 
+For example, to compile your program with gcc using x64 debug wxWidgets DLLs
+you would need to use the following options for the compiler (and `windres`
+resource compiler):
+
+    -I$WXWIN/include -I$WXWIN/lib/gcc_x64_dll/mswud -D__WXMSW__ -DWXUSINGDLL
```

Comment:
IME, when building wxWidgets with the bundled GCC makefile, `_x64` in the library folder name is not used (unlike with MSVC), i.e., both 32- and 64-bit libraries are by default produced in `$WXWIN/lib/gcc_{lib|dll}` folder.

`_x64` in the folder name is used when building with CMake or with official pre-built binaries.

I would expect people using the makefile have wxWidgets also built by the makefile. I think it would be easiest to just remove "x64" from the text and `_x64`  from the examples?

Reply:
You're absolutely right, thanks for noticing this. I'll fix it and push.